src/libs/API/types.ts

```diff
@@ -169,7 +169,7 @@ const WRITE_COMMANDS = {
     DELETE_POLICY_REPORT_FIELD: 'DeletePolicyReportField',
     SET_POLICY_TAGS_REQUIRED: 'SetPolicyTagsRequired',
     SET_POLICY_REQUIRES_TAG: 'SetPolicyRequiresTag',
-    RENAME_POLICY_TAG_LIST: 'RenamePolicyTaglist',
+    RENAME_POLICY_TAG_LIST: 'renamePolicyTagList',
```

Comment:
```suggestion
    RENAME_POLICY_TAG_LIST: 'RenamePolicyTaglist',
```

This is the name of the API command so we need to keep it the same